pom.xml

```diff
@@ -176,4 +220,147 @@
+                                <phase>clean</phase>
+                                <goals>
+                                    <goal>clean</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.jreleaser</groupId>
+                        <artifactId>jreleaser-maven-plugin</artifactId>
+                        <inherited>false</inherited>
+                        <configuration>
+                            <jreleaser>
+                                <release>
+                                    <github>
+                                        <owner>mcollovati</owner>
+                                        <name>quarkus-hilla</name>
+                                        <changelog>
+                                            <formatted>ALWAYS</formatted>
+                                            <preset>conventional-commits</preset>
+                                            <skipMergeCommits>true</skipMergeCommits>
+                                            <hide>
+                                                <categories>
+                                                    <category>merge</category>
+                                                    <category>tasks</category>
+                                                    <category>build</category>
+                                                </categories>
+                                                <contributors>
+                                                    <contributor>GitHub</contributor>
+                                                </contributors>
+                                            </hide>
+                                            <contributors>
+                                                <format>- {{contributorName}}</format>
+                                            </contributors>
+                                        </changelog>
+                                        <draft>true</draft>
+                                        <tagName>${project.version}</tagName>
+                                        <releaseName>Release ${project.version}</releaseName>
+                                        <prerelease>
+                                            <pattern>.*\.(alpha|beta|rc)\d+</pattern>
+                                        </prerelease>
+                                    </github>
+                                </release>
+                                <deploy>
+                                    <maven>
+                                        <artifactory>
+                                            <app>
+                                                <active>${release.jfrog.active}</active>
+                                                <url>https://mcollovati.jfrog.io/artifactory/jreleaser-tests</url>
```

Comment:
Wrong repo

Reply:
Good catch. I have to configure also jfrog with the correct name

Reply:
done